tslib/src/functions-ps-report/common/ps-report-service-bus-messages.ts

```diff
@@ -5,6 +5,7 @@ export interface PsReportListSchoolsIncomingMessage {
   requestedBy: string
   dateTimeRequested: string
   jobUuid: string
+  urns?: number[]
```

Comment:
or undefined?

Reply:
that's what the question mark is for

Reply:
I think the question mark means the property may be omitted.  But in the definition in the admin app the property may be there with an undefined value.

Reply:
yeah good point, i will update this